docs/fluxible-context.md

```diff
@@ -53,6 +53,8 @@ This action context object is used every time an `executeAction` method is calle
 
 Generates a context interface providing access to only the functions that should be called from components. By default: `executeAction`, `getStore`. `executeAction` does not allow passing a callback from components so that it enforces actions to be send and forget.
 
+Note: You may provide an app level `componentActionHandler` function when instantiating Fluxible. This allows you to catch errors (at a high level) spawning from components firing actions.
```

Comment:
We could even just link to the fluxible.io docs once they are up for use cases. Adding associated docs to the Controller View guide.